pipeline-templates/job/code-build.yml

```diff
@@ -67,8 +67,8 @@ jobs:
     displayName: 'Copy Files to: $(build.artifactstagingdirectory)'
     inputs:
       Contents: |
-        azure\**
-        src\${{ parameters.SolutionBaseName }}.Database\projects\**
+        azure/**
+        src/${{ parameters.SolutionBaseName }}.Database\projects\**
```

Comment:
Missing some forward slashes at the end

Reply:
Slashes are not needed at the end as wildcarding and will match our github template